Lib9c/Action/BalanceDoesNotExistsException.cs

```diff
@@ -1,4 +1,5 @@
 using Libplanet;
+using Libplanet.Assets;
```

Comment:
```suggestion
```

Reply:
It's needed because `Currency` was moved from `Libplanet` to `Libplanet.Assets`.